src/seq_analysis/orf.rs

```diff
@@ -93,16 +93,18 @@ pub struct Orf {
 
 /// The current algorithm state.
 struct State {
-    start_pos: [Option<usize>; 3],
+    start_pos: [Vec<usize>; 3],
     codon: VecDeque<u8>,
+    found: VecDeque<Orf>,
```

Comment:
If more than one ORF is found after `.next()` is called, then they are added to a queue and popped on each subsequent call of `.next()` before sequence scanning resumes